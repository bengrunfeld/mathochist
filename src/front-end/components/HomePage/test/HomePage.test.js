import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import { MemoryRouter } from 'react-router-dom'

import HomePage from '../HomePage'


describe('<HomePage />', () => {
  test('matches the snapshot', () => {
    const tree = mount(<HomePage />)
    expect(tree).toMatchSnapshot()
  })
})
