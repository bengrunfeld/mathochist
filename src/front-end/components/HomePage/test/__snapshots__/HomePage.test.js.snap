// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> matches the snapshot 1`] = `
<HomePage>
  <div>
    <h1>
      Home Page
    </h1>
  </div>
</HomePage>
`;
